src/NServiceBus.AcceptanceTests/NServiceBus.AcceptanceTests.csproj

```diff
@@ -27,6 +28,7 @@
     <WarningLevel>4</WarningLevel>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <Prefer32Bit>false</Prefer32Bit>
+    <NoWarn>PCR0001</NoWarn>
```

Comment:
Pretty sure we discussed this in the past and came to the conclusion that ConfigureAwait would be just noise in the tests?(perhaps we should revisit this)